protoc-gen-gogo/generator/generator.go

```diff
@@ -840,9 +840,10 @@ func (g *Generator) SetPackageNames() {
 	// Register the support package names. They might collide with the
 	// name of a package we import.
 	g.Pkg = map[string]string{
-		"fmt":   RegisterUniquePackageName("fmt", nil),
-		"math":  RegisterUniquePackageName("math", nil),
-		"proto": RegisterUniquePackageName("proto", nil),
+		"fmt":          RegisterUniquePackageName("fmt", nil),
+		"math":         RegisterUniquePackageName("math", nil),
+		"proto":        RegisterUniquePackageName("proto", nil),
+		"golang_proto": RegisterUniquePackageName("golang_proto", nil),
```

Comment:
The imports need to be printed.  You probably need to add an import print for golang_proto here?
```
if gogoproto.ImportsGoGoProto(g.file.FileDescriptorProto) {
		g.PrintImport(g.Pkg["proto"], g.ImportPrefix+"github.com/gogo/protobuf/proto")
	} else {
		g.PrintImport(g.Pkg["proto"], g.ImportPrefix+"github.com/golang/protobuf/proto")
	}
	g.PrintImport(g.Pkg["fmt"], "fmt")
	g.PrintImport(g.Pkg["math"], "math")
```